packages/amplication-data-service-generator/scripts/generate-test-data-service.ts

```diff
@@ -31,6 +47,10 @@ export default async function generateTestDataService(
     pluginInstallations: installedPlugins,
   });
   await writeModules(modules, destination);
+  await copyFiles(
+    ADMIN_GLOB_SOURCES,
+    path.join(__dirname, "..", `${destination}/admin-ui/public/`)
+  );
```

Comment:
we should not copy the files manually in the test. The test simulates a full process and we expect the files to be included in the results.

Is there a reason you needed to add this code? The way I see it, it should be enough to add the files to the static/public folders, like you already did.

Reply:
Earlier I thought as well all the files should be copied automatically once added into public folder but Reason I'd added this code because static assets were not copied through existing code and I'd opened issue for same as well #4040.

Reply:
@yuval-hazaz did you get a chance to look at my last comment :) ?

Reply:
@shubhwip thanks for sharing. we will look into this soon

Reply:
@shubhwip Can you please respond to Yuval's review and check if deleting the code (by any chance) still gives the same result?

Reply:
I'd already commented :) https://github.com/amplication/amplication/pull/4041#discussion_r986460121 above but I can confirm again that if we only put the static assets in public folder and generate a sample service with current generate sample app logic it will not copy static assets correctly instead it will create files with similar names with incorrect encoding which is an issue mentioned in #4040.

Reply:
@shubhwip I looked again at what you wrote and just realized that we have an issue that will not allow us to fix #3963 with local files because the way the DSG is currently working will not allow us to copy encoded files into the generated app. 
So, to fix #3963 I added the missing files in the original URL 
https://amplication.com/assets/react-admin.png
https://amplication.com/assets/graphql.png
https://amplication.com/assets/restapi.png

We will need to fix #4040 separately and only then we will be able to use local assets

So, thank you very much for your support and for opening #4040 
I will close this PR and issue now